atomic_dbus.py

```diff
@@ -374,6 +378,40 @@ def _ScanSetup(self, scan_targets, scanner, scan_type, rootfs, _all, images, con
     def Scan(self, scan_targets, scanner, scan_type, rootfs, _all, images, containers):
         return self._ScanSetup(scan_targets, scanner, scan_type, rootfs, _all, images, containers).scan()
 
+    @slip.dbus.polkit.require_auth("org.atomic.read")
+    @dbus.service.method("org.atomic", in_signature='ssss', out_signature= 's')
+    # sudo busctl --system call org.atomic /org/atomic/object org.atomic ActiveScans
+    # Only support scanning one image at a time via the async method
+    # Same with rootfs
+    def ScanAsync(self, scan_id, scanner=None, scan_type=None, rootfs=None):
```

Comment:
The named arguments represent optional arguments.  So, in this case, if you accept the defaults, you don't have to pass anything.  This points out bug that you must provide either scan_id or rootfs. I'll fix that prior to merge.

As for None, you don't have to pass anything there so I think the lack of None is moot.  We can always tweak this after the fact if you prefer.